packages/experiments/src/components/ExtendedPicker/examples/ExtendedPeoplePicker.Basic.Example.tsx

```diff
@@ -100,10 +105,15 @@ export class ExtendedPeoplePickerTypesExample extends BaseComponent<{}, IPeopleP
           'aria-label': 'People Picker'
         } }
         componentRef={ this._setComponentRef }
+        headerComponent={ this._renderHeader() }
       />
     );
   }
 
+  private _renderHeader(): JSX.Element {
+    return <div>TO:</div>;
```

Comment:
Did you intend for this to just be `TO:`?

Reply:
Since this is just the example, yes this was just a simple element, to demonstrate an element rendering before the selected contents. I can make this more robust if you prefer.

Reply:
Oh shoot! I completely missed that it was an example, disregard my comment.

Reply:
@joschect did you have any other comments?